src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+
+      {age && <p className="Person__age">I am {age}</p>}
+
+      <p className="Person__partner">
+        {isMarried ? `${partnerName} is my ${partner}` : 'I am not married'}
```

Comment:
change it for better readability. Use this syntax all the time in the future.
```suggestion
        {
          isMarried 
            ? `${partnerName} is my ${partner}`
            : 'I am not married'
         }
```